managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -1746,7 +1746,7 @@ public Result healthCheck(UUID customerUUID, UUID universeUUID) {
     try {
       List<HealthCheck> checks = HealthCheck.getAll(universeUUID);
```

Comment:
Should this be getAllOrBadRequest? Will that get rid of the runtime exception catching below?

Reply:
Not sure, thereis no unit tests also.